tests/FixtureBuilder/Denormalizer/Fixture/SpecificationBagDenormalizer/Value/UniqueValueDenormalizerTest.php

```diff
@@ -83,7 +83,7 @@ public function testReturnsUniqueValueIfUniqueFlagsFound()
         $result = $denormalizer->denormalize($fixture, $flags, $value);
 
         $this->assertInstanceOf(UniqueValue::class, $result);
-        $this->stringContains('dummy_id', $result->getId());
+        $this->assertStringStartsWith('Dummy#', $result->getId());
```

Comment:
Was `stringContains` use in purpose?

Reply:
I'm fine with changing this if this is a issue I don't think there was a strong reasons for it.